packages/tools/u-boot/package.mk

```diff
@@ -50,7 +53,8 @@ make_target() {
     echo "see './scripts/uboot_helper' for more information"
   else
     [ "${BUILD_WITH_DEBUG}" = "yes" ] && PKG_DEBUG=1 || PKG_DEBUG=0
-    [ -n "$UBOOT_FIRMWARE" ] && find_file_path bootloader/firmware && . ${FOUND_PATH}
+    [ -n "$ATF_PLATFORM" ] && cp -av $(get_install_dir atf)/usr/share/bootloader/bl31.bin .
```

Comment:
Why are you changing ATF handling? I think this is out of scope for this PR. AFAIK it was made this way to support other projects when they start using open source replacements but I have no idea how far they are with that...

Reply:
It's a distinct patch, see https://github.com/LibreELEC/LibreELEC.tv/pull/4762/commits/2504672c37ab753d008e9a32cd908da93f20470d
We need some crust stuff in the u-boot package anyway (to express the dependency when required), so I figured we can move that one line too. Now it's all in one place, and that atf handling in not sunxi specific anyway.

Reply:
@lrusak @CvH are you fine with this?

Reply:
I am going to have to revert this change in #4591 since there is more files that needs to be copied on rockchip, see https://github.com/LibreELEC/LibreELEC.tv/pull/4591/files#diff-1cdbf9d1405af32af934d0e88882b5449c0fdc93a55b7f646f7f21583185ff1e

Reply:
Oh alright, I wasn't aware that RK requires additional files